components/PostChooser/editor-partials/modal/index.js

```diff
@@ -25,6 +22,7 @@ export const PostChooserModal = ( props ) => {
 		postTypes,
```

Comment:
```suggestion
		postTypes = [
			{ label: __( 'Posts' ), value: 'post' },
			{ label: __( 'Pages' ), value: 'page' },
		], // Default post types to search.
```

Since this `postTypes` array is used here for the selectedPostType useState(), I'd maybe define the default value here instead of in postchooser.js. Postchooser.js can just pass whatever is set in the block and still override this.